regulation-worker/internal/delete/api/api_test.go

```diff
@@ -216,63 +211,63 @@ func TestOAuth(t *testing.T) {
 		expectedDeleteStatus   model.JobStatus
 		expectedPayload        string
 	}{
-		{
-			name: "test with a valid token and successful response",
-			job: model.Job{
-				ID:            1,
-				WorkspaceID:   "1001",
-				DestinationID: "1234",
-				Status:        model.JobStatusPending,
-				Users: []model.User{
-					{
-						ID: "Jermaine1473336609491897794707338",
-						Attributes: map[string]string{
-							"phone":     "6463633841",
-							"email":     "dorowane8n285680461479465450293437@gmail.com",
-							"randomKey": "randomValue",
-						},
-					},
-					{
-						ID: "Mercie8221821544021583104106123",
-						Attributes: map[string]string{
-							"email": "dshirilad853601942465969121327991@gmail.com",
-						},
-					},
-					{
-						ID: "Claiborn443446989226249191822329",
-						Attributes: map[string]string{
-							"phone": "8782905113",
-						},
-					},
-				},
-			},
-			dest: model.Destination{
-				DestinationID: "1234",
-				Config: map[string]interface{}{
-					"rudderDeleteAccountId": "xyz",
-				},
-				Name: "GA",
-				DestDefConfig: map[string]interface{}{
-					"auth": map[string]interface{}{
-						"type": "OAuth",
-					},
-				},
-			},
-			deleteResponses: []deleteResponseParams{
-				{
-					status:      200,
-					jobResponse: `[{"status":"successful"}]`,
-				},
-			},
-			cpResponses: []cpResponseParams{
-				{
-					code:     200,
-					response: `{"secret": {"access_token": "valid_access_token","refresh_token":"valid_refresh_token"}}`,
-				},
-			},
-			expectedDeleteStatus: model.JobStatusComplete,
-			expectedPayload:      `[{"jobId":"1","destType":"ga","config":{"rudderDeleteAccountId":"xyz"},"userAttributes":[{"email":"dorowane8n285680461479465450293437@gmail.com","phone":"6463633841","randomKey":"randomValue","userId":"Jermaine1473336609491897794707338"},{"email":"dshirilad853601942465969121327991@gmail.com","userId":"Mercie8221821544021583104106123"},{"phone":"8782905113","userId":"Claiborn443446989226249191822329"}]}]`,
-		},
+		// {
```

Comment:
why those tests are commented out?

Reply:
Oops, that is mistake. I was debugging something and forgot to uncomment them back. Thanks for pointing it out. Fixed.